ztp/siteconfig-generator/siteConfig/siteConfig_test.go

```diff
@@ -74,6 +74,32 @@ spec:
 	assert.Equal(t, siteConfig.Spec.Clusters[0].Nodes[3].Role, "master")
 }
 
+// Test cases for default/specific values on SiteConfig.Clusters[].ClusterImageSetNameRef
+
+func TestClusterImageSetNameRef(t *testing.T) {
```

Comment:
Would you add unit test in siteConfigBuilder_test.go making sure the AgentClusterInstall for both clusters getting different versions.

Reply:
Done.